src/translations/en.json

```diff
@@ -7163,6 +7163,7 @@
               "description": "The To-do list card allows you to add, edit, check-off, and remove items from your to-do list.",
               "integration_not_loaded": "This card requires the `todo` integration to be set up.",
               "hide_completed": "Hide completed items",
+              "hide_create": "Hide add item option",
```

Comment:
```suggestion
              "hide_create": "Hide add item",
```

Reply:
Reading `<verb> <verb> <noun>` throws me off a bit. Maybe something more like `Hide 'Add item' field` is a little clearer?

Reply:
Yes, I was not sure about the best wording from the start. Thank you for the suggestion!